### DISCUSSION TITLE
MIOpen Windows Support for GFX906 GPUs

### DISCUSSION THREAD

**Discussion by ENDlezZenith** [ROCmSoftwarePlatform/MIOpen#2614] [General]
I have two AMD Radeon Pro VII GPUs with the Infinity Fabric Link connected. while I saw the official documentation stated that GFX906 GPUs are entering maintenance mode (as stated in https://rocm.docs.amd.com/en/latest/about/release-notes.html). Will GFX906 GPUs be able to use ROCm + MIOpen + PyTorch in Windows once the support of MIOpen for Windows is released or this function would not be provided to GFX906 GPUs?
Thank you!

**Comment by atamazov** [comment]
@Na3MnO4 
> Will GFX906 GPUs be able to use ROCm + MIOpen + PyTorch in Windows once the support of MIOpen for Windows is released?

Currently this is not supported. MIOpen _supports_ gfx906, but ROCm runtime for Windows doesn't, please see https://rocm.docs.amd.com/projects/install-on-windows/en/latest/reference/system-requirements.html.
